core/types/transaction.go

```diff
@@ -219,13 +247,14 @@ func (tx *Transaction) Size() common.StorageSize {
 // XXX Rename message to something less arbitrary?
```

Comment:
Below you'll notice I got confused about what these `message`s are.  Since there is a note here already in geth to consider renaming this, and we are adding a new "message" keyword in this PR, we might want to consider it. xD

Reply:
Not sure how this is used, just updated it since it is Transaction-like. Even if poorly named, we should probably keep parity with Geth (👀) on anything that we don't have a good reason to change or pulling in updates from Geth will be even hairier than it would otherwise be.